src/Symfony/Component/Console/Command/CompleteCommand.php

```diff
@@ -82,8 +71,7 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         try {
-            // "symfony" must be kept for compat with the shell scripts generated by Symfony Console 5.4 - 6.1
-            $version = $input->getOption('symfony') ? '1' : $input->getOption('api-version');
+            $version = $input->getOption('api-version');
```

Comment:
What about this one @wouterj? should we keep support for the deprecated flag to provide a smooth transition?
Also, we're missing a deprecation notice if this should be removed in 7.0

Reply:
This was modified by #46901.

We need to keep the `symfony` option if we want developers to be able use to bash completion for projects with different Symfony versions on the same computer. The completion script in `symfony/console` from 5.4.x to 6.1.x sets the option `-S` but not `-a`.
https://github.com/symfony/symfony/blob/a6e5b75ecdf1266ab019fbfd7042a5d6efab0c04/src/Symfony/Component/Console/Resources/completion.bash#L28

We can fallback to `api-version=1` when not specified.
```suggestion
            $version = $input->getOption('api-version', 1);
```

It would be better to make this change in a dedicated PR.

Reply:
If I understand correctly, here is what you suggest:
- Create a PR targeting 6.4 as a new feature, making `api-version` fallback to `1`
- Update `symfony` option deprecation to be removed in 8.0, when 5.4 is EOLed.

I'll take care of this if it is what you meant

Reply:
I think we should revert this change completely and leave it as-is. Notice the comment says "kept for compat" without deprecating the old one, which is intentional.

Shell completion scripts are global and there is no reason to make life extremely difficult for developers that work on several Symfony versions, some of which are older than 7.0, for the gain of removing these 2 LOC :)

Reply:
This suits me fine. As you said, these are just 2 lines of code :smile: I reverted this change

Reply:
Do I have something to do with this?